packages/lodestar/src/chain/factory/attestation/index.ts

```diff
@@ -1,4 +1,4 @@
-import {BeaconBlock, BeaconState, IndexedAttestation, Shard, Slot} from "../../../types";
+import {BeaconBlock, BeaconState, IndexedAttestation, Shard, Slot} from "../../../../types";
```

Comment:
all this imports should be  `--- from "@chainsafe/eth2-types"`

Reply:
Yeah im still debuggin this stuff.... idk why its like this